### DISCUSSION TITLE
Dairy interface overhaul

### DISCUSSION THREAD

**Discussion by RKroebel** [holos-aafc/Holos#189] [Ideas]
The current dairy interface structure implies that dairy works like beef (a for all female cattle common birth time and lactation period) rather than staggering pregnancy and lactation for a steady amount of lactating vs. dry cows throughout the year to guarantee a consistent milk output. 
QC LL lait carbonneutre has requested that we re-design the dairy interface to represent the above, and they are willing to help with the design (contact E. Charbonneau).

**Comment by sarahp1980** [comment]
**Based on a meeting between the Holos team and E. Charbonneau, F. Nadon and S. Binggeli in Oct 2024, we can consider future modifications to the Holos dairy interface as follows:**
**1)** Have a default setup where all animal groups are present on the farm year-round, i.e., lactating cows, dry cows, heifers and calves are present 365 days a year, with the number of animals in each group remaining (relatively) constant;
**2)** For lactating dairy cows, we should retain three distinct management periods as a default, as diet/management for these animals can vary throughout the year;
**3)** Add an input field where the user can specify "Days in milk (DIM)";
**4)** We can consider changing our setup for dairy calves - typically, in a dairy system, all male calves and any female calves not required as replacement heifers are removed from the cow after 7-10 days. Female calves to be kept as replacement heifers are removed from the cow after approx. 1 day and fed milk replacer; these animals are weaned at ~56 days old after which they are moved to a forage diet (partial or full forage?). We will need to determine if we have separate dairy calf groups for unweaned female calves up to 56 days old that are on a milk (replace)-only diet and for female calves > 56 days of age that are on a forage diet but not yet considered replacement heifers;
**5)** We can consider adjusting the interface to account for the movement of dairy cattle from barn to pasture and vice versa within a single day. Currently, if a user wants to account for differing emissions that occur when the animals are in the barn vs on pasture, they could do this by having separate groups of dairy animals with differing numbers, to represent (approximately) the proportion of time spend inside/outside during a typical day for these groups. In Holos, this could be achieved by adding input fields where the user can specify if the animals split their time between different housing systems in a single day, and if so, how much time during the day they spend in each system and the diet and manure management practices in each housing system;
**6)** We do not currently directly consider the amount of water added to manure in dairy systems, and how this affects the C and N concentration of the manure and its volume. As we estimate manure volume based on the estimated amount of N excreted by these animals and literature-derived manure N concentration and moisture content values for different manure types, it is possible that these literature-derived concentration values do not always accurately reflect the manure composition on an individual farm, possibly leading to mismatches between the Holos-estimated manure volume and the farm-reported manure volume. The user could address this by altering the manure moisture content and/or N concentration values for manure;
**7)** We can update the enteric CH4 methodology in V6 to incorporate equations from the NASEM (2021) - Roland will discuss this with C. Benchaar;
**8)** We could consider improving our estimates of manure (C&N?) production to base it on milk production rather than GEI?;
**9)** Although we have added an entry to the FAQ to advise model users on how to represent multiple cuts of hay or silage in a single field in a single year, this does not address the impacts of such practices on fuel/energy consumption - in a future version, we can try to have fuel/energy CO2 factors that are more reflective of the number of operations on the farm.
